pkg/minishift/update/update_test.go

```diff
@@ -84,15 +89,73 @@ func TestExtractBinary(t *testing.T) {
 		extName = "zip"
 		osName = "windows"
 	}
+	archiveName := fmt.Sprintf("minishift-%s-%s-%s.%s", "1.0.0", osName, "amd64", extName)
+	testArchivePath := filepath.Join(testDataDir, archiveName)
+	tmpArchivePath := filepath.Join(testDir, archiveName)
+	// Copy test archive to tmpDir for testing purpose
+	copyArchive(t, testArchivePath, tmpArchivePath)
+	fileName, err := extractBinary(tmpArchivePath, testDir)
+	t.Log(fileName)
```

Comment:
I am not sure whether we should add more noise to the test output. Generally we should aim to have as little was possible get mixed into the test output.